results.js

```diff
@@ -0,0 +1,33 @@
+const results = JSON.parse(localStorage.getItem('votes'));
```

Comment:
This `'votes'` key doesn't seem to match the key that you set in you `appp.js`. You don't get the data to render here until you change that.